managed-databases/document-database/quickstart.mdx

```diff
@@ -59,11 +59,12 @@ This product is currently only a prototype in a **private testing phase**. You c
 
 ### Via database client
 
-1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
 2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
-3. Click on **Download TLS certificate** and store the file where you will execute the connection command.
-4. Copy the connection command located under **Connection** in the Database Instance information page.
-5. Paste the connection command in your terminal, and edit the username field with the login you chose when creating the database. The full command should look like the following:
+3. Click <Icon name="download" /> next to the **TLS certificate** to download it.
+4. Transfer the file to where you will execute the connection command.
+5. Copy the connection command located under **Connection** in the Database Instance information page.
+6. Paste the connection command in your terminal, and edit the username field with the login you chose when creating the database. The full command should look like the following:
     ```bash
     mongosh --tls --tlsCAFile rdb-databasename.pem 'mongodb://username@ip:port/databasename?authMechanism=PLAIN'
```

Comment:
since "rdb-" prefix was relevant (it has been changed to "docdb" now in the automatically suggested database instance name), it has been removed from the downloaded certificate file name.
Moreover, the certificate name is not the database name, but the "database instance" name (the issue already existed before these modifications).  I don't know if difference between database instance and database is clear to user, maybe we can emphasize that the database instance is the resource created in Console and might look like "docdb-xxxxx-xxxx" if created automatically.
```suggestion
    mongosh --tls --tlsCAFile databaseinstancename.pem 'mongodb://username@ip:port/databasename?authMechanism=PLAIN'
```